resources/zib/zib-MultidisciplinaryTeamMeeting.xml

```diff
@@ -0,0 +1,196 @@
+      </slicing>
+    </element>
+    <element id="Encounter.participant.individual.extension:patient">
+      <path value="Encounter.participant.individual.extension" />
+      <sliceName value="patient" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Encounter-PatientParticipant" />
+      </type>
+    </element>
+    <element id="Encounter.participant.individual.extension:patient.value[x]">
+      <path value="Encounter.participant.individual.extension.value[x]" />
+      <short value="Patient" />
+      <definition value="The patient as participant of the patient consultation." />
+      <alias value="Patient" />
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.16" />
+        <comment value="Patient" />
+      </mapping>
+    </element>
+    <element id="Encounter.period">
+      <path value="Encounter.period" />
+      <comment value="'PatientConsultationDateTime' is mapped to both `period.start` and `period.end`. When all is known is a single value, both `.start` and `.end` should be populated with this value, otherwise the `period`, and thus the MultidisciplinaryTeamMetting, is considered 'ongoing'. In FHIR R4, the Period datatype has an invariant that inconsistently says in text that Period.start shall be lower than Period.end, but actually supports lower than or equal in its expression, which reflects the original intent. This has been fixed for FHIR R5; the text now reflects the expression. &#xD;&#xA;&#xD;&#xA;When a system is able to populate both `period.start` and `period.end`, the value of `.start` is leading when converting FHIR data to a zib instance." />
+    </element>
+    <element id="Encounter.period.start">
+      <path value="Encounter.period.start" />
+      <short value="PatientConsultationDateTime" />
+      <definition value="The date (and time) the patient consultation is scheduled or took place." />
+      <alias value="PatientbesprekingDatumTijd" />
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.6" />
+        <comment value="PatientConsultationDateTime" />
+      </mapping>
+    </element>
+    <element id="Encounter.period.end">
+      <path value="Encounter.period.end" />
+      <short value="PatientConsultationDateTime" />
+      <definition value="The date (and time) the patient consultation is scheduled or took place." />
+      <alias value="PatientbesprekingDatumTijd" />
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.6" />
+        <comment value="PatientConsultationDateTime" />
+      </mapping>
+    </element>
+    <element id="Encounter.reasonReference">
+      <path value="Encounter.reasonReference" />
+      <slicing>
+        <discriminator>
+          <type value="profile" />
```

Comment:
I am wondering if type="profile" would be needed here. This would be a costly operation for a validator. Perhaps type would already be sufficient?